test/minikube/minikube_base_database_test.go

```diff
@@ -183,27 +183,20 @@ func verifyPacketFetch(t *testing.T, namespaceName string, admin0 string) {
 	t.Log("tar contents: ", output)
 }
 
-func verifyEngineAltAddress(t *testing.T, namespaceName string, admin0 string, expectedNrEngines int) {
+func verifyEngineAltAddress(t *testing.T, namespaceName string, adminPod string, expectedNrEngines int) {
 	kubectlOptions := k8s.NewKubectlOptions("", "", namespaceName)
 
-	podNames, err := k8s.RunKubectlAndGetOutputE(t, kubectlOptions, "exec", admin0, "--",
-		"bash", "-c",
-		"nuocmd get processes | grep -o \"(address=[^/]*\"| cut -f2 -d'='")
-	require.NoError(t, err)
-	podNamesSlice := strings.Split(strings.TrimSuffix(podNames, "\n"), "\n")
-
-	actualAltAddresses, err := k8s.RunKubectlAndGetOutputE(t, kubectlOptions, "exec", admin0, "--",
-		"bash", "-c",
-		"nuocmd get processes | grep -o \"alt-address: [^}]*\" | cut -f2 -d' '")
-	require.NoError(t, err)
-	actualAltAddressesSlice := strings.Split(strings.TrimSuffix(actualAltAddresses, "\n"), "\n")
+	output, err := k8s.RunKubectlAndGetOutputE(t, kubectlOptions, "exec", adminPod, "--",
+		"nuocmd", "--show-json", "get", "processes", "--db-name", "demo")
+	require.NoError(t, err, output)
 
-	require.True(t, len(podNamesSlice) == expectedNrEngines, "Expected number of process names don't match")
-	require.True(t, len(actualAltAddressesSlice) == expectedNrEngines, "Expected number of process addresses don't match")
+	err, objects := testlib.Unmarshal(output)
```

Comment:
How does the parser handle multiple JSON object in the same string, e.g.
```
{
  "startId": "0",
  ...
}
{
  "startId": "1",
  ...
}
```

Reply:
Go does not care. It continues unmarshalling the string until it runs out of top-level objects. We have tests for that. It can handle this form of JSON.